source/scripts/Dude.js

```diff
@@ -14,8 +14,8 @@ export default class Dude extends Pixi.Sprite {
         this.anchor.x = 0.5
         this.anchor.y = 0.5
         
-        this.position.x = Math.random() * 320
-        this.position.y = Math.random() * 240
+        this.position.x = 640 / 2
```

Comment:
Magic numbers. Granted, they were magic before this change too, so it's not a net loss.

Reply:
Also not reflected in the commit comments.